content/pt-br/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -0,0 +1,389 @@
+na seção abaixo.
+
+### 2.a. Elimine campos puramente mutantes {#eliminate-mutating-fields}
+
+Se a `PodSecurityPolicy` está mudando os pods, então você pode eliminar os pods que 
+não satisfazem os requisitos do nível de segurança do Pod, quando você finalmente 
+desativar a `PodSecurityPolicy`. Para evitar isso, você deve eliminar todo PSP mutante 
+antes da troca. Infelizmente o PSP não separa de maneira clara os campos mutantes 
+e de validação, portanto, esta não é uma migração direta.
+
+Você pode começar eliminando os campos que são puramente mutantes e não têm nenhuma 
+influência na política de validação. Esses campos de referência (também listados no 
+[Mapeando Padrões de Segurança `PodSecurityPolicies` ao Pod]
+(/docs/reference/access-authn-authz/psp-to-pod-security-standards/)) são:
+
+- `.spec.defaultAllowPrivilegeEscalation`
+- `.spec.runtimeClass.defaultRuntimeClassName`
+- `.metadata.annotations['seccomp.security.alpha.kubernetes.io/defaultProfileName']`
+- `.metadata.annotations['apparmor.security.beta.kubernetes.io/defaultProfileName']`
+- `.spec.defaultAddCapabilities` - Embora tecnicamente um campo de mutação e validação, 
+   devam ser fundidos em `.spec.allowedCapabilities` 
+   que executa a mesma validação sem realizar mutações.
+
+{{< caution >}}
+Remover isso pode resultar em cargas de trabalho perdendo configuração necessária, 
+e causar problemas. Veja [Implementando as políticas atualizadas](#psp-update-rollout) 
+abaixo, para obter sugestões sobre como implementar essas mudanças com segurança.
+{{< /caution >}}
+
+### 2.b. Elimine as opções não cobertas pelos padrões de segurança 
+de pod {#eliminate-non-standard-options}
+
+Existem vários campos em `PodSecurityPolicy` que não são cobertos pelos padrões 
+de segurança de Pod. Se você precisa aplicar essas opções, você precisará complementar 
+a admissão de segurança de Pod com um
+[webhook de admissão](/docs/reference/access-authn-authz/extensible-admission-controllers/),
+que está fora do escopo deste guia.
+
+Primeiro, você pode remover os campos puramente validadores que não cobrem os 
+padrões de segurança de Pod. Esses campos (também listados no
+[Mapeando de Padrões de Segurança `PodSecurityPolicies` ao Pod](/docs/reference/access-authn-authz/psp-to-pod-security-standards/)
+referenciados por "no opinion") que são:
+
+- `.spec.allowedHostPaths`
+- `.spec.allowedFlexVolumes`
+- `.spec.allowedCSIDrivers`
+- `.spec.forbiddenSysctls`
+- `.spec.runtimeClass`
+
+Você também pode remover os seguintes campos, relacionados a grupos 
+de contrôles POSIX / UNIX.
+
+{{< caution >}}
+Se algum deles usar a estratégia `MustRunAs` eles podem estar mudando! 
+Remover isso pode resultar em cargas de trabalho, não definindo corretamente 
+os grupos necessários, e causar problemas. Veja [Implantando as políticas atualizadas]
+(#psp-update-rollout) abaixo para obter sugestões sobre como implementar 
```

Comment:
This link "Implantando as políticas atualizadas" is broken. Remove the newline after the brackspaces.